DependencyInjection/Configuration.php

```diff
@@ -84,19 +84,22 @@ public function getConfigTreeBuilder()
                                 ->arrayNode('publishable')
                                     ->addDefaultsIfNotSet()
                                     ->children()
-                                        ->scalarNode('form_group')->defaultValue('form.group_publish_workflow')->end()
+                                        ->scalarNode('form_tab')->defaultValue('form.tab_publish')->end()
+                                        ->scalarNode('form_group')->defaultValue('group_publish_workflow')->end()
                                     ->end()
                                 ->end()
                                 ->arrayNode('publish_time')
                                     ->addDefaultsIfNotSet()
                                     ->children()
-                                        ->scalarNode('form_group')->defaultValue('form.group_publish_workflow')->end()
+                                        ->scalarNode('form_tab')->defaultValue('form.tab_publish')->end()
+                                        ->scalarNode('form_group')->defaultValue('group_publish_workflow')->end()
```

Comment:
why no longer form.group_publish_workflow? this will break translation overwrites for example.